vendor/golang.org/x/tools/internal/typeparams/common.go

```diff
@@ -87,7 +87,6 @@ func IsTypeParam(t types.Type) bool {
 func OriginMethod(fn *types.Func) *types.Func {
 	recv := fn.Type().(*types.Signature).Recv()
 	if recv == nil {
-
 		return fn
 	}
 	base := recv.Type()
```

Comment:
Based on the provided code patch, the only change is removing an empty statement which has no impact on the code's functionality or its correctness.

However, without more context or information about the program being developed, it's difficult to provide more detailed feedback or identify any potential risks or improvements.